### ISSUE TITLE
Integration (at least Segment) anonymousIds differ from analytics.js anonymousId

### ISSUE THREAD

**Issue by tommedema** [DavidWells/analytics#11]
For data analysis reasons it is important for the anonymousId to be the same across integrations. We're having difficulties analysing our tracking data because we expected the following two IDs to be the same:

![Screen Shot 2019-07-25 at 3 23 41 PM](https://user-images.githubusercontent.com/331833/61912992-95caf100-aef0-11e9-8c75-57154554e9b0.png)

Do you agree that each integration should retrieve the analytics.js anonymousId if already known when loading? For now we're going to work around this by either by telling segment what analytics.js' anonymousId is or by telling analytics.js what Segment's anonymous ID is, tbd.

And if so, would the same be necessary for the userId?

**Comment by tommedema** [comment]
Fyi current workaround (ugly):

```
      // Sync the analytics.js library's anonymousId with Segment's anoymousId
      const segment = window.analytics
      if (segment) {
        const segmentUser = segment.user()
        if (segmentUser) {
          const segmentAnonymousId = segmentUser.anonymousId()
          if (segmentAnonymousId) {
            a.setAnonymousId(segmentAnonymousId)
          }
        }
      }
```

**Comment by DavidWells** [comment]
Fixed via https://github.com/DavidWells/analytics/pull/13

The `syncAnonymousId` option was added. It needs to be set to `true`. 

Next major release it will probably be on by default.

```
const analytics = Analytics({
  debug: true,
  plugins: [
	segmentPlugin({
	  writeKey: 'your-key',
	  syncAnonymousId: true,
	})
  ]
})
```

**Comment by DavidWells** [comment]
Released with `analytics-plugin-segment@0.2.0`
